redisvl/index/index.py

```diff
@@ -255,8 +229,7 @@ class SearchIndex(BaseSearchIndex):
         from redisvl.index import SearchIndex
 
         # initialize the index object with schema from file
-        index = SearchIndex.from_yaml("schemas/schema.yaml")
-        index.connect(redis_url="redis://localhost:6379")
+        index = SearchIndex.from_yaml("schemas/schema.yaml", redis_url="redis://localhost:6379")
```

Comment:
Can we add a few more examples of init to the docstring here since this has changed? Ditto for the async class